low-level/files/opt/seagate/sspl/setup/sspl_post_install.py

```diff
@@ -151,8 +166,19 @@ def validate_dependencies(setup):
         pkg_validator = PkgV()
         pkg_validator.validate_pip3_pkgs(host=socket.getfqdn(),
             pkgs=pip3_3ps_packages_main, skip_version_check=False)
-        pkg_validator.validate_rpm_pkgs(host=socket.getfqdn(),
-            pkgs=rpm_3ps_packages, skip_version_check=False)
+        os_release_version = None
+        with open("/etc/os-release") as f:
+            os_release = f.read()
+            os_release_version = \
+                re.search("VERSION_ID=\"(.*)\"\n", os_release).group(1)
```

Comment:
I can reduce this code to get version_id as `get_os_info()['VERSION_ID']`, if we merge main with rocky_linux.. It is written by Satish with manifest PR.

Reply:
rocky linux branch is updated now, optimize the code

Reply:
updated 👍